src/components/app/pageAdvocatesHeatmap/advocatesHeatmapTooltip.tsx

```diff
@@ -46,18 +46,20 @@ export function TotalAdvocatesPerStateTooltip({
   // Calculate the adjusted position so it does not overflow the window
   const centeredX = mousePosition.x - offsetX
   const adjustedX = Math.min(Math.max(centeredX, 0), window.innerWidth - tooltipWidth)
+  const formattedNumber = `${FormattedNumber({ amount: totalAdvocatesPerState, locale })} advocates`
+  const tooltipWidthBasedOnTextLength = formattedNumber.length * 10
 
   return (
     <div
-      className="pointer-events-none fixed z-50 flex h-[46px] w-[193px] items-center justify-center rounded-2xl bg-black px-1 font-sans text-base text-white"
+      className={`pointer-events-none fixed z-50 flex h-[46px] w-[${tooltipWidthBasedOnTextLength}px] items-center justify-center rounded-2xl bg-black px-4 font-sans text-base text-white`}
```

Comment:
np: what do you think about using cn here?

Like this:
```suggestion
      className={cn('pointer-events-none fixed z-50 flex h-[46px] items-center justify-center rounded-2xl bg-black px-4 font-sans text-base text-white', `w-[${tooltipWidthBasedOnTextLength}px]`)}
```